### PR TITLE
Docsoverview

### PR DESCRIPTION
**Pull request by jelmerdereus**
Extend the documentation overview  page. Add slack to the homepage and docs and community page.

README.md
docs/sources/overview.md
docs/theme/home.html
docs/theme/community.html

### CODE REVIEW

**Comment by jelmerdereus** [comment]
@samertm please have a look

File: README.md [hunk 1]
```diff
@@ -69,3 +69,10 @@ srclib by installing an editor plugin in your editor of choice. [See all editor
 ## License
 Sourcegraph is licensed under the [MIT License](https://tldrlegal.com/license/mit-license).
 More information in the LICENSE file.
+
+
+## Contributing
+
+**In case you want to get involved and start hacking on srclib bugs and features or write your own toolchain, [sign up for Slack](http://slackin.srclib.org/) and access it on  [srclib.slack.com](http://srclib.slack.com) .**
```
Comment (shurcooL) [inline README.md]: I wouldn't describe the slackin.srclib.org link as "sign up for Slack", as that's something you would do on Slack.com. slackin.srclib.org is more like "get invited to the srclib Slack team".

Comment (jelmerdereus) [inline README.md]: Thanks for your feedback @shurcooL I changed it

**Comment by jelmerdereus** [comment]
This will resolve issue #122

File: README.md (hunk 1 above)
Comment (beyang) [inline README.md]: @jelmerdereus I think @shurcooL was saying change the text to be something like:

```
If you want to start hacking on srclib or write your own srclib toolchain, [join the srclib Slack](slackin.srclib.org) and then access it on [srclib.slack.com](http://srclib.slack.com).
```

**Comment by beyang** [comment]
This looks good to merge, pending the minor change in language. I'll just merge it now, and you can update that later.

**Pull request merged**